moengage/build.gradle

```diff
@@ -21,11 +20,22 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    compileOnly 'androidx.annotation:annotation:1.1.0'
+    implementation 'androidx.annotation:annotation:1.5.0'
 
     // rudder core sdk
-    compileOnly 'com.rudderstack.android.sdk:core:1.+'
-    api 'com.moengage:moe-android-sdk:10.5.00'
+    implementation 'com.rudderstack.android.sdk:core:[1.0,2.0)'
```

Comment:
Haven't we decided to base it on minor version?

Reply:
Didn't get it completely, can you please explain a bit more about what and when we decided?
Also, what should be the version?

Reply:
[1.7.0, 1.8.0)